cmd/main.go

```diff
@@ -100,10 +109,8 @@ func buildCmdTree() *cobra.Command {
 	createCmd := newCreateCmd()
 	// kubebuilder create api
 	createCmd.AddCommand(newAPICmd())
-	// kubebuilder create webhook (v2 only)
```

Comment:
Do not remove the whole comment, if you want to remove the v2 only part I'm ok with that

Reply:
has no need for the comment any more since it has not v1 and v12 webhooks.

Reply:
Remove the `(v2 only)` part, but leave the `// kubebuilder create webhook` as there are comments documenting all commands, if you see two lineas above you have `// kubebuilder create api`

Reply:
Done